pre_commit/languages/helpers.py

```diff
@@ -18,7 +18,7 @@
 if TYPE_CHECKING:
     from typing import NoReturn
 
-FIXED_RANDOM_SEED = 1542676186
+FIXED_RANDOM_SEED = 1542676187
```

Comment:
I changed this to keep a few 50/50 tests passing -- the switch from `random=` to not _also_ changed the ordering